WikipediaCleaner/src/org/wikipediacleaner/api/constants/Contributions.java

```diff
@@ -81,7 +81,7 @@ public String getDescription() {
       return buffer.toString();
     }
 
-    return GT._T("You haven''t done any contribution on \"{0}\"", wikipedia.toString());
+    return GT._T("You haven't done any contribution on \"{0}\"", wikipedia.toString());

```

Comment:
Thanks for the proposed fix, but the doubled single quote is currently necessary for strings that are translated with a variable part (the {0} here), otherwise it prevents translation to work correctly.

And the result in English is still correct (because they're cleaned up before using them)
![image](https://user-images.githubusercontent.com/3530686/222828079-2093ab38-b774-495b-8a45-90f59268db99.png)